### ISSUE TITLE
Should we update our streamflow notebook to write directly to S3 or replaces with new NWM rechunking?

### ISSUE THREAD

**Issue by amsnyder** [hytest-org/hytest#4]
Notebook 1 of the streamflow evaluation workflow currently writes to a local file. Should we update it to write directly to S3?

I think notebook 1 will be replaced by my rechunking of the NWM data. Maybe we should modify notebook 2 to read from this new file once it is created.

**Comment by rsignell-usgs** [comment]
@amsnyder , yes, great idea to write directly to s3.     I've modified this notebook to write a subset to `s3://nhgf-develoment/workspace`: 
https://jupyter.qhub.esipfed.org/hub/user-redirect/lab/tree/shared/users/rsignell/repos/hytest-development/gallery/conus404_subset_to_zarr.ipynb

**Comment by gzt5142** [comment]
Success in writing in parallel direct to S3 in the OSN appliance. 

The demo/tutorial for streamflow does just that (see the notebook at <https://github.com/hytest-org/hytest/blob/main/model_evaluation/tutorials/nwm_streamflow/01_Data_Prep.ipynb>)

I had some challenges in getting the cluster workers authenticated correctly to write to the OSN bucket. In the end it worked fine, so long as I did not trust `ebdpy.py` to do anything.  It snuck in extra environment variables which confused the botocore profile/credentials API. 

--gt
